bindings/csharp/Regorus/Regorus.csproj

```diff
@@ -0,0 +1,62 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.Regorus</RootNamespace>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Version>0.4.0-preview.1</Version>
+    <LangVersion>10.0</LangVersion>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+  </PropertyGroup>
+
+
+  <!-- Determine build os/arch -->
+  <PropertyGroup>
+    <IsLinuxX64>false</IsLinuxX64>
+    <IsWindowsArm64>false</IsWindowsArm64>
+    <IsWindowsX64>false</IsWindowsX64>
+    <IsOSXArm64>false</IsOSXArm64>
+
+    <IsLinuxX64 Condition="$([MSBuild]::IsOsPlatform('Linux')) And $([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture) == X64">true</IsLinuxX64>
```

Comment:
Isn't this going to vary the config depending on which platform/architecture that the **build machine** is running?

I'd have thought you'd want consitent results wherever the build runs to get either:
* Single nuget package with multiple architecture libaries bundled inside and a means of resolving the correct one
* A nuget package per platform architecture

Reply:
The original intent was to support both a pipleine build for all supported targets and also a local build for only the current host platform for local development workflows. The host-target-triple was being figured out was so that `cargo build -r --target <host-target-triple>` can be invoked to generate the regorus ffi shared library in `target/<host-target-triple>/release`, just as the pack expects. If host-target-triple was not passed to cargo, then the shared library will be generated in `target/release` and won't get picked up by pack.

I have removed the local workflow completely to simpify the csproj.